README.md

```diff
@@ -1,12 +1,16 @@
-[Codecov][1] Ruby Example
-=========================
+# [Codecov][1] Ruby Example
+## Guide
+### Travis Setup
 
-> Example of how to integrate with [Codecov.io][1] for your **awesome** Ruby project!
+Add to your `.travis.yml` file.
+```yml
+lang: ruby
 
-## See this repos [Coverage Reports][4]
+script: bundle exec rake
 
-
-## Usage
+rvm: 2.1
+```
+### Produce Coverage Reports
```

Comment:
IMO this section is what confuses me the most between this repo and https://github.com/codecov/codecov-ruby. 

@TomPed what are your thoughts on changing this title to be "How this repo produces coverage reports" ?